### ISSUE TITLE
Accessibility: add aria tags to new mode selector

### ISSUE THREAD

**Issue by fpurcell** [opentripplanner/otp-ui#315]
- adding an aria-label to the (i) button

- the middle area buttons (things like Uber vs P&R, or the list of e-scooter toggle buttons) lack the aria-checked attribute

- some of these middle area button elements might work better in a screen reader with the role="radio"
 
(related to TORA issues: PUBWEB-956, PUBWEB-954 and PUBWEB-955, respectively)

<img width="525" alt="Screen Shot 2021-11-16 at 10 40 46 AM" src="https://user-images.githubusercontent.com/64323/142702904-5683ac4e-68d4-4230-98f8-97c1697006d5.png">

<img width="624" alt="Screen Shot 2021-11-16 at 11 10 57 AM" src="https://user-images.githubusercontent.com/64323/142702901-80a91704-de15-41d6-b268-ff3fb594b773.png">

<img width="357" alt="Screen Shot 2021-11-16 at 11 11 49 AM" src="https://user-images.githubusercontent.com/64323/142702897-1a5bbbcc-4f65-4f92-9878-7fee37d00a31.png">
